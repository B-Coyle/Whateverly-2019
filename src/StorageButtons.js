import React, {Component} from 'react';
import App from './App.js';

class StorageButtons extends App {
  constructor() {
    super();

  }
  render {
    return (
      
    )
  }
}

export default StorageButton;