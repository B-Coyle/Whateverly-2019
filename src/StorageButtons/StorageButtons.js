import React, {Component} from 'react';
import App from '../App/App.js';

export default class HikePopUpCard extends Component  {
  constructor() {
    super();
    

    //EL: favorite button to display (toggle)
    //EL: visited button to display (toggle)
    //EL: reset search button 

    //method:
    //display favorites
    
    //method:
    //display visited

    //method:
    //reset buttons in input fields
  }
  render () {
    return (
      <section>
        <input type='button' value='Favorite' />
        <input type='button' value='Visited' />
        <input type='button' value='Reset' />

      </section>
      //favorite

      //visited
      //reset search  
    )
  }
}
