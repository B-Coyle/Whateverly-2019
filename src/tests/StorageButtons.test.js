import React from 'react';
import ReactDom from 'react-dom';
import StorageButtons from '../StorageButtons.js';
import { shallow } from 'enzyme';

const mockCamping = [
    {
    "name": "Glacier Basin Campground",
    "location": "Estes Park",
    "season": "May-Sept",
    "elevation": 8500,
    "camping": ["Tent", "Car", "RV"],
    "activities": ["Hiking", "Fishing", "Horseback Riding", "Wildlife"],
    "reservation-available": true,
    "nearest-city": null
      },
      {
    "name": "Grand Junction KOA",
    "location": "Grand Junction",
    "season": "All year",
    "elevation": 4600,
    "camping": ["Tent", "Car", "RV"],
    "activities": ["ATVing"],
    "reservation-available": true,
    "nearest-city": null
    }
]

describe('StorageButton', () => {
let wrapper 

beforeEach(() => {
    wrapper.shallow(
        <StorageButtons />
    )
});

it('should mock the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
});

})