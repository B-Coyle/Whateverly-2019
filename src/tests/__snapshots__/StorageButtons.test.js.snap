// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StorageButton should mock the snapshot 1`] = `
<section>
  <input
    type="button"
    value="Favorite"
  />
  <input
    type="button"
    value="Visited"
  />
  <input
    type="button"
    value="Reset"
  />
</section>
`;
